### ISSUE TITLE
Decieve not showing up as active

### ISSUE THREAD

**Issue by 404Tiny** [molenzwiebel/Deceive#216]
I've been using it for some time now and had no problem, but now whenever I use decieve to open up valorant, I no longer see "decieve active" on my friendslist.

does anybody have a solution for this?

**Comment by molenzwiebel** [comment]
Yes, a new change to VALORANT seems to have moved the fake Deceive user into the "offline" section of the friends list. Deceive still works. I'm working on a fix, but I'll keep this open until its fixed.

**Comment by herumes** [comment]
> Yes, a new change to VALORANT seems to have moved the fake Deceive user into the "offline" section of the friends list. Deceive still works. I'm working on a fix, but I'll keep this open until its fixed.

Any update on this?
Btw, for me it doesn't show in the friends list at all... even in the "offline" section of the friends list.
